e107_languages/English/admin/lan_admin.php

```diff
@@ -513,5 +513,6 @@
 define("LAN_LOCK", "Lock");
 define("LAN_SYSTEM_USER", "System user");
 define("LAN_ENGINE", "Engine");
+define("LAN_SOURCE", "Source");
 
```

Comment:
Thank you. These entries need to be replaced with LAN_SOURCE now. 

lan_admin_log.php
define("RL_LAN_098", "Source");

lan_mailout.php
define("LAN_MAILOUT_133", "Source");

lan_users.php
define("USRLAN_197", "Source");